DNN Platform/Library/Services/FileSystem/FileVersionInfo.cs

```diff
@@ -54,7 +54,7 @@ public FileVersionInfo()
         public string Extension { get; set; }
         
         [XmlElement("size")]
-        public int Size { get; set; }
+        public long Size { get; set; }
```

Comment:
This creates backward compatibility issue. Must deprecate and create another one.